packages/itwin/grouping-mapping-widget/src/components/SharedComponents/DeleteModal.tsx

```diff
@@ -9,16 +9,16 @@ import {
   ModalButtonBar,
   Text,
 } from "@itwin/itwinui-react";
-import React, { useState } from "react";
-import { handleError } from "../../common/utils";
+import { useMutation } from "@tanstack/react-query";
+import React, { useCallback, useState } from "react";
 import "./DeleteModal.scss";
 import { LoadingSpinner } from "./LoadingSpinner";
 
 export interface DeleteModalProps {
   entityName?: string;
```

Comment:
Can the `entityName` ever be `undefined`?

Reply:
Nope. Use to but not anymore.